devel/python/python/ert_gui/ertwidgets/checklist.py

```diff
@@ -0,0 +1,154 @@
+from PyQt4.QtCore import QSize, Qt
+from PyQt4.QtGui import QToolButton, QHBoxLayout, QLabel, QListWidget, QWidget, QVBoxLayout, QListWidgetItem, QMenu, QAction, QAbstractItemView, QListView
+
+from ert_gui.ertwidgets import addHelpToWidget
+from ert_gui.widgets.search_box import SearchBox
+from ert_gui.widgets.util import resourceIcon
+
+
+class CheckList(QWidget):
```

Comment:
This could essentially be a `QListView`.  See [QListView.setModel](http://pyqt.sourceforge.net/Docs/PyQt4/qabstractitemview.html#setModel).